src/main/java/com/getbase/jenkins/plugins/metrics/history/influxdb/generators/JenkinsBasePointGenerator.java

```diff
@@ -43,6 +70,7 @@ public JenkinsBasePointGenerator(Run<?, ?> build) {
         final Integer resultInt = resultStr.equals("SUCCESS") ? 1 : 0;
         final String jobAbsoluteURL = build.getParent().getAbsoluteUrl();
         final String buildAbsoluteURL = jobAbsoluteURL + build.getNumber();
+        final Map<String, Object> fields = getBuildParameters(build);
```

Comment:
Please rename `fields` to something more meaningful - maybe just `buildParameters`?

Reply:
@karolgil fixed